### DISCUSSION TITLE
New version 3.116.1

### DISCUSSION THREAD

**Discussion by fox-15** [mono/SkiaSharp#3105] [Q&A]
Good day!

After the update, warnings appeared warning CS0618

```cs
static SKSize MeasureText(string text, int size)
{
    var paint = new SKPaint
    {
        Color = SKColors.Black,
        TextSize = size,                 <-- warning CS0618 Use SKFont.Size instead.
        TextAlign = SKTextAlign.Center   <-- warning CS0618 Use SKTextAlign method overloads instead.
    };
    var width = paint.MeasureText(text); <-- warning CS0618 Use SKFont.MeasureText() instead.
    return new SKSize(width, size);
}
```

The API Documentation is available on the web to browse.
But there version SkiaSharp 2.88.x

Please explain how this can be changed?

Thank you!

**Comment by dmitryvhf** [comment]
> Good day!
> 
> After the update, warnings appeared warning CS0618

Similar discussion: https://github.com/mono/SkiaSharp/discussions/3104
Maybe this help you.

**Comment by artemiusgreat** [comment]
You need to create separate objects for SKFont, SKTextAlign, etc, and pass them as an extra argument to Draw functions. 
https://github.com/Indemos/Canvas/commit/820c876755bd4c3ab4dd3b92416a36bc65523648#diff-5a7e648dc67c91aaab97993e871482f5b179f99d94068e946c886530f0cb5ab6R259
